t/plugin/limit-conn.t

```diff
@@ -956,3 +956,216 @@ GET /test_concurrency
 200
 --- no_error_log
 [error]
+
+
+
+=== TEST 25: create consumer and bind key-auth plugin
+--- config 
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                    "username": "consumer_jack",
+                    "plugins": {
+                        "key-auth": {
+                            "key": "auth-jack"
+                        }
+                    }
+                }]]
+                )
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 26: create route and consumer_name is consumer_jack
```

Comment:
bad title.

it should be `create route and enable plugin 'key-auth'`

Reply:
done